org.eclipse.mylyn.context.core/META-INF/MANIFEST.MF

```diff
@@ -6,8 +6,11 @@ Bundle-Version: 3.26.0.qualifier
 Bundle-Activator: org.eclipse.mylyn.internal.context.core.ContextCorePlugin
 Bundle-Localization: plugin
 Require-Bundle: org.eclipse.core.runtime,
+ org.eclipse.ui,
```

Comment:
The UI dependency is not acceptable for "core" bundle

Reply:
new commit is now in place.